src/main/java/bio/terra/janitor/service/cleanup/FlightSubmissionFactoryImpl.java

```diff
@@ -48,6 +48,8 @@ public FlightSubmission createSubmission(TrackedResource trackedResource) {
         return FlightSubmission.create(AzureContainerInstanceCleanupFlight.class, flightMap);
       case TERRA_WORKSPACE:
```

Comment:
Once we start to assign managed identities, we need to update this code to remove assignments of MIs to workspace resources as assignments are not deleted automatically upon MI deletion and will be dangling as Unknown references. We need a task or mention for this clean up job.

Reply:
Hm true, MI deletion in Janitor _could_ also find+remove all assignments. However VMs should also be cleaned up by Janitor. Maybe once we have some connected tests in WSM that assign VMs to MIs we can observe how well Janitor works.